src/formats/color-mode-attributes.js

```diff
@@ -19,39 +34,80 @@ function colorModeAttributes({dictionary: origDictionary, file, options}) {
         name: `${token.name}-hsl`,
         path: [...token.path, 'hsl']
       })
-      acc.push({...token, value: convertToHSL(token.value), darkValue: convertToHSL(token.darkValue)})
+
+      /**
+       * Creates a new token object adjacent to the original
+       * This takes the raw HSL values and wraps it in a HSL css function
+       */
+      const tokenWithHSL = {...token, value: convertToHSL(token.value)}
+
+      /**
+       * Checks if the token object contains one of the supported modes
+       * If yes; mutate the HSL-specific token object with the converted HSL value for the alternate mode
+       */
+      for (const mode of supportedModes) {
+        if (token.hasOwnProperty(mode)) {
+          tokenWithHSL[mode] = convertToHSL(token[mode])
+        }
+      }
+
+      acc.push(tokenWithHSL)
```

Comment:
What is the output here?

Is it:

```js
{
  value: // hsl,
  dark: // hsl for dark
}
```

If it is, is there a possibility to nest into value?

```js
{
  value: {
    light: // hsl light,
    dark: // hsl dark
  }
}
```

This would bring us closer to the w3c standard. Or should this rather be done in a separate PR?

Reply:
This is a slightly deeper refactor and unrelated to the goal of this PR.. so will defer to a future PR if that's okay. The tricky part will be disentangling the default behavior of StyleDictionary, which requires a `value` property to always be present.

Reply:
I like the idea though, thanks @lukasoppermann 🙇